examples/dsource/main.tf

```diff
@@ -0,0 +1,112 @@
+/**
+* Summary: This template showcases the properties available when creating an app data dsource.
+*/
+
+terraform {
+  required_providers {
+    delphix = {
+      version = "VERSION"
+      source  = "delphix-integrations/delphix"
+    }
+  }
+}
+
+provider "delphix" {
+  tls_insecure_skip = true
+  key               = "1.XXXX"
+  host              = "HOSTNAME"
+}
+
+
+# resource "delphix_appdata_dsource" "test_app_data_dsource" {
+#   source_id                  = "1-APPDATA_STAGED_SOURCE_CONFIG-6"
+#   group_id                   = "1-GROUP-1"
+#   log_sync_enabled           = false
+#   make_current_account_owner = true
+#   link_type                  = "AppDataStaged"
+#   name                       = "appdata_dsource"
+#   staging_mount_base         = ""
+#   environment_user           = "HOST_USER-2"
+#   staging_environment        = "1-UNIX_HOST_ENVIRONMENT-2"
+#   parameters = jsonencode({
+#     externalBackup : [],
+#     delphixInitiatedBackupFlag : true,
+#     delphixInitiatedBackup : [
+#       {
+#         userName : "delphix",
```

Comment:
nit: I think we could provide username and userpass as "xxx" rather than "delphix". It is just for consistency with the other examples.

Reply:
agreed